pkg/utils/values_patch.go

```diff
@@ -243,16 +285,25 @@ func CompactPatches(operations []*ValuesPatchOperation) ValuesPatch {
 	return newValuesPatch
 }
 
+type ApplyPatchOptions string
+
+const Strict ApplyPatchOptions = "strict"
+
 // ApplyValuesPatch applies a set of json patch operations to the values and returns a result
-func ApplyValuesPatch(values Values, valuesPatch ValuesPatch) (Values, bool, error) {
+func ApplyValuesPatch(values Values, valuesPatch ValuesPatch, options ...ApplyPatchOptions) (Values, bool, error) {
```

Comment:
Could we avoid optional arguments here?

Reply:
"options" arguments are replaced with the "mode" argument and two modes are defined: Strict and IgnoreNonExistentPaths.